documentation/config-reference.md

```diff
@@ -23,53 +23,49 @@ Information on properties in `config.js`:
 
 - **`deviceType`** is the Device Under Test (DUT) attached to the testbot. Example: Raspberrypi3 64-bit device, the `deviceType` property needs to be `raspberrypi3-64`.
 - **`suite`** is the absolute path to the test suite directory that you want to execute. The path after `__dirname` in the config is a path relative to the workspace directory.
-- **`networkWired`** and **`networkWireless`** properties for configuration of Network Manager. Used to configure the right mode of connection for the DUT to connect to the Access Point (AP) created by the testbot.
-- **`balenaApiKey`** is the balenaCloud API key used when running the release suite. Ideally, you can add it as an environment variable and reference it with `process.env.BALENACLOUD_API_KEY`.
+- **`networkWired`** and **`networkWireless`** properties are configuration for the Network Manager. This sets up the Access Point (AP) created by the testbot for the DUT to use while provisioning.
+- **`balenaApiKey`** is the balenaCloud API key used when running the cloud release suite. Ideally, you can add it as an environment variable and refere it with `process.env.BALENACLOUD_API_KEY`.
 - **`balenaApiUrl`** is the balenaCloud environment you are targetting for your test suite. Production is `'balena-cloud.com'` and staging is `'balena-staging.com'`.
-- **`organization`** is the balenaCloud organizations where test applications are created. Ideally, you can add it as an environment variable and reference it with `process.env.BALENACLOUD_ORG`.
-- **`image`** is the absolute path to the balenaOS image that is flashed onto the Device Under Test (DUT). The image should be kept under the `workspace` directory. The path after `__dirname` in the config is a path relative to the workspace directory. Make sure to rename the image to balena.img. If you provide `balena.img` as your balenaOS image, then Leviathan will compress it for you in `gz` format. We recommend compressing beforehand, as it saves time. 
+- **`organization`** is the balenaCloud organizations where test applications are created for the cloud suite.
+- **`image`** is the absolute path to the balenaOS image that is flashed onto the Device Under Test (DUT). The image should be kept under the `workspace` directory. The path after `__dirname` in the config is a path relative to the workspace directory. 
```

Comment:
@lmbarros You can provide any path you intend to provide. This is more of a recommendation that everything can be part of the workspace directory since there are gitignore rules in place to ignore those images

Reply:
Added your explanation